src/sentry/monitors/utils.py

```diff
@@ -18,3 +21,64 @@ def signal_first_monitor_created(project: Project, user, from_upsert: bool):
         first_cron_monitor_created.send_robust(
             project=project, user=user, from_upsert=from_upsert, sender=Project
         )
+
+
+def get_alert_rule(project: Project, user: User, monitor: Monitor, alert_rule: dict):
+    """
+    Gets a dict formatted alert rule to create alongside the monitor
+    :param project: Project object
+    :param user: User object that made the request
+    :param monitor: Monitor object being created
+    :param alert_rule: dict of information, must contain alert targets
```

Comment:
So what goes in the dictionary, what kind of information??

Reply:
Ah right this gets auto-documented